.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release
+on:
+  push:
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+    permissions:
+      contents: write
```

Comment:
@superbrothers 
I don't have the settings tab on the GitHub repo's page. Can you see the default permission of `secrets.GITHUB_TOKEN`?

Reply:
It probably looks like it has write, but even if it changes in the future, it will still work with this setting, so I think it's fine as it is.